lib/errors.js

```diff
@@ -23,7 +23,7 @@ function StatusCodeError(statusCode, body, options, response) {
 
     this.name = 'StatusCodeError';
     this.statusCode = statusCode;
-    this.message = statusCode + ' - ' + (JSON && JSON.stringify ? JSON.stringify(body) : body);
+    this.message = 'Request status code was not 2xx';
```

Comment:
👍 @gurpreetatwal . Sorry for being so unresponsive lately. ...am really busy. I’ll take time to review this PR hopefully soon after your push.